docs/Kubernetes/kubernetes.md

```diff
@@ -0,0 +1,70 @@
+### Introduction
+Note: Not for production use!!
+This guide with provide detail on how to deploy the AMTCloudToolkit services in Microsoft(C) Azure(R).
+Functionality has been added to MPS to allow it to scale and support a greater number of devices. 
+For this deployment, kubernetes will be used running in Azure(R) along with utilizing both redis and consul.
+
+### High level Design
+
+####Figure #1 MPS Scaling Architecture
+[![Scaling architechure](../assets/images/ScallingHighLevel.png)](../assets/images/ScallingHighLevel.png)
+Figure # 1 shows a high-level diagram of the overall architechture of the MPS scaling mode.
+    
+1. Devices connect to an available MPS Server through the load balancer.
+1. The REST api requests are routed to an available Web Server (a component of MPS running in scale mode) though a load balancer.
+1. The Web Server determines which MPS Server to route the traffic to based on which MPS Server the device is connected to and sends that traffic through the MPS Proxy connection. 
+1. The MPS Server sends the traffic to the corresponding device.
+
+### MPS Configuration
+To support running the service in a distributed environment, some configuration settings were added to MPS.
```

Comment:
OK, so where are these things happening? Might we say anything about this?

Reply:
You could also put these in a table. Would it make it more readable? I'm asking, not telling.